library/ix-dev/community/audiobookshelf/README.md

```diff
@@ -0,0 +1,8 @@
+# Audiobookshelf
+
+[Audiobookshelf](https://www.audiobookshelf.org/) is a self-hosted audiobook and podcast server.
+
+> When application is installed, a container will be launched with **root** privileges.
+> This is required in order to apply the correct permissions to the `Audiobookshelf` directories.
+> Afterward, the `Audiobookshelf` container will run as a **non**-root user (Default: `568`).
+> All mounted storage(s) will be `chown`ed only if the parent directory does not match the configured user.
```

Comment:
```suggestion
> The following applies only when the storage type is set to **ixVolume**
> When application is installed, a container will be launched with **root** privileges.
> This is required in order to apply the correct permissions to the `Audiobookshelf` directories.
> Afterward, the `Audiobookshelf` container will run as a **non**-root user (Default: `568`).
> All mounted storage(s) will be `chown`ed only if the parent directory does not match the configured user.
```

Update the other readme too please!
> now that reminds me that I should update the rest of the apps to reflect that!

Reply:
Ah, explains that now too!  I've only used hostPath mapping so far and this snippet concerned me at first but I've come to really appreciate rootless containers.  Learned a TON these last 3 months setting up a new TrueNAS Scale machine!